tests/functional/TestSmokeTest/tests/heketi_test.go

```diff
@@ -284,7 +308,7 @@ func HeketiCreateVolumeWithGid(t *testing.T) {
 		"sudo useradd writer2 -G writegroup -p'$6$WBG5yf03$3DvyE41cicXEZDW.HDeJg3S4oEoELqKWoS/n6l28vorNxhIlcBe2SLQFDhqq6.Pq'",
 		fmt.Sprintf("sudo mount -t glusterfs %v /mnt", volInfo.Mount.GlusterFS.MountPoint),
 	}
-	_, err = vagrantexec.ConnectAndExec("192.168.10.100:22", cmd, 10, true)
+	_, err = vagrantexec.ConnectAndExec(storage0+":"+portNum, cmd, 10, true)
```

Comment:
Would it be nicer to have a predefined storage0ssh variable that already includes the port in order to avoid having to repeat the string concatenation for the port ? (ditto for storage1,2,3,etc)

Reply:
Good suggestion!